install/CPU/windows_cpu.bat

```diff
@@ -2,6 +2,6 @@
 cd ..
 cd ..
 py -3.10 -m venv .venv
-call .venv\Scripts\activate
-pip install -r requirements.txt
-pip install torch==2.0.1 torchvision==0.15.2 torchaudio==2.0.2
+SET VenvPythonPath=%CD%\.venv\Scripts\python.exe
```

Comment:
We should use `setlocal` and `endlocal` to not save this VenvPythonPath forever.
Or reset the SET at the end? But the requirements install may crash and the VenvPythonPath will still be set.

Reply:
good idea